README.md

```diff
@@ -18,23 +18,24 @@ The [**chkware**](https://chkware.github.io/) (pronounced as /check:ware/) exten
 
 ## Quick start
 
-Get started writing `chkware` configurations with VS Code doing following steps:
+Get started writing _**CHKware**_ configurations with VS Code doing following steps:
 
-- Step 1: If you haven't done so already, install [chkware](https://chkware.github.io/setup#install-chkware-toolset-with-pipx)
+- Step 1: If you haven't done so already, install [_**CHKware**_](https://chkware.github.io/setup) by following this link.
 
-- Step 2: Install the `chkware` extension for VS Code
+- Step 2: Search and install the _**CHKware**_ extension for VS Code.
 
-New to `chkwere`? Read the [_quick start_](https://chkware.github.io/quick-start) guide here.
+New to CHKwere? Read the [_quick start_](https://chkware.github.io/quick-start) guide here.
 
 ## Setup development environment
```

Comment:
This might confuse the general users that they need to do this when using the extension. We should indicate that this is for those who want to contribute. Or move the section elsewhere.

Reply:
For now let's keep it like this, I am going to update the documentation to be more inclusive to chkware vsc extension next. I understand your reasoning.